pkgs/development/embedded/avrdude/default.nix

```diff
@@ -16,14 +14,20 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-pGjOefWnf11kG/zFGwYGet1OjAhKsULNGgh6vqvIQ7c=";
   };
 
-  nativeBuildInputs = [ cmake bison flex ];
+  nativeBuildInputs = [
+    cmake
+    bison
+    flex
+    util-linux
```

Comment:
Instead of using `util-linux` everywhere, the better option is to pick the binaries from `unixtools`.